### PR TITLE
fix(tag): [MAG-000] bumping tag

### PR DESCRIPTION
**Pull request by philjonasfiltered**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@excelwithbusiness/webmobile-sc-components",
-  "version": "1.27.2",
+  "version": "1.27.4",
```
Comment (eTheMan80) [inline package.json:3]: @philjonasfiltered do you need to manually update the components library?

Comment (philjonasfiltered) [reply package.json:3]: in general I don't, but the GitHub actions didn't detect my changes 🤷

Comment (tomaisthorpe) [reply package.json:3]: It didn't detect changes because of the squash

**Review by tomaisthorpe — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (philjonasfiltered) [reply package.json:3]: ah, I'll keep that in mind. Thanks 👍

**Pull request merged**